src/Transcriber.js

```diff
@@ -59,22 +61,24 @@ class Transcriber extends EventEmitter {
     return this._initPromise;
   }
 
-  start() {
+  start({audioParams = {}, relayParams = {}} = {}) {
     if (this.state !== READY) {
       throw new Error('Transcriber#start: not ready');
     }
 
     this.state = INITIALIZING_RECORDING;
 
     const sessionId = uuid();
+    const socketAudioParams = Object.assign({}, this.audioParams, audioParams);
+    const socketRelayParams = Object.assign({}, this.relayParams, relayParams);
```

Comment:
can always use spread operator for these

Reply:
Good call, updated in 8f5643c.